src/test/scala/fable/ConsumerSpec.scala

```diff
@@ -231,5 +233,45 @@ class ConsumerSpec extends AsyncFunSuite {
       producer.close
     }
 
+  private def createMockKafkaObjects = {
+    val topic = Topic("fable-test-example")
+    val offset = 1.toLong
+    val partitions: ArrayBuffer[TopicPartition] = ArrayBuffer[TopicPartition]()
+    val topicPartition: TopicPartition = new TopicPartition(topic.name, 1)
+    partitions += topicPartition
+    val partitionMap =
+      Map(topicPartition -> offset.asInstanceOf[java.lang.Long]).asJava
+
+    val mockConsumer =
+      new MockConsumer[String, String](OffsetResetStrategy.EARLIEST)
+
+    mockConsumer.subscribe(Seq(topic.name).asJava)
```

Comment:
I'm confused about why we'd call this here. Don't we expect `fable.Consumer` to call `subscribe`?

Reply:
So we did forget to take a subscribe out of the test body but we can't take out this subscribe. For some reason when we try to it throws a `Attempt to dynamically assign partitions while manual assignment in use`

Reply:
That's interesting. Is it throwing an `IllegalStateException`? I know you're not allowed to both manually assign partitions via `assign` and use auto-balancing via `subscribe` in the same consumer, but it doesn't seem like you're doing that and I would expect an extra `subscribe` to make it worse and not better, so...I'm confused!

I wouldn't hold up the merge on this if it's working, but if you can't figure it out, I'll try to poke at it tomorrow.

Reply:
Right! It's very bewildering. I wonder if it's because we're updating the offsets? But it seems like we also have to update the offsets otherwise we can't poll.